public/pages/ReviewAndCreate/components/DetectorDefinitionFields/DetectorDefinitionFields.tsx

```diff
@@ -94,19 +94,36 @@ export const DetectorDefinitionFields = (
         !props.validDetectorSettings &&
         props.validationResponse.hasOwnProperty('message')
       ) {
-        return (
-          <EuiCallOut
-            title="Issues found in the detector settings"
-            color="danger"
-            iconType="alert"
-            size="s"
-            style={{ marginBottom: '10px' }}
-          >
-            <ul>
-              <li>{props.validationResponse.message}</li>
-            </ul>
-          </EuiCallOut>
-        );
+        if (isEqual(props.validationResponse.validationType, 'model')) {
```

Comment:
`props.validationResponse` is listed as an optional field. To prevent an NPE, suggest to change `props.validationResponse.validationType` -> `get(props, 'validationResponse.validationType', '')`